static/carousel-spotlight/js/index.js

```diff
@@ -26,15 +26,36 @@ var initialize = function(){
         }, '*');
     };
 
+    var analytics = new Analytics();
+    analytics.initConfig({
+        logUrl: settings.api_base_url + '/__tvpa.gif',
+        domain: Utils.isset(location, 'hostname') ? location.hostname : '',
+        firstPartyCookies: settings.firstpartycookies,
+        cookieDomain: settings.cookiedomain,
+        loginId: settings.loginid,
+    });
+    analytics.track('ci', {li: settings.loginid});
+
     Carousel(name, carouselSettings);
 };
 
+
+var loadJS = function(src){
+    var script = document.createElement("script");
+    script.type = "text/javascript";
+    script.src = src;
+    document.body.appendChild(script);
+};
+
+loadJS((document.body.getAttribute('data-domain')+'carousel-spotlight/js/libs/analytics.js'));
+loadJS("https://a.tvpage.com/tvpa.min.js");
```

Comment:
The tvpa.min.js library is already included at iframe creation and is loaded asynchronously, there's no need to do this.